modules/standard/Socket.md

```diff
@@ -0,0 +1,82 @@
+## Socket Module API
```

Comment:
I have defined `TCPConn` as a translation of `UDPSocket` because the socket isn't just in an unconnected state but will have a established connection and also helps to differ from the TCP socket one we get from `listen`